.github/workflows/test-ios-simulator.yaml

```diff
@@ -72,6 +72,7 @@ jobs:
             --exclude integration_test/service_cellular_test.dart \
             --exclude integration_test/service_wifi_test.dart \
             --exclude integration_test/swipe_test.dart \
+            --exclude integration_test/webview_leancode_test.dart \
```

Comment:
what happens with it? flaky?

Reply:
I've removed this test from `exclude` because I wanted to see failed test for workflow testing purposes. Webview tests don't work on simulators

Reply:
Ah, okay.